CHANGELOG.md

```diff
@@ -39,6 +39,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Solve a panic when deserializing an empty BSON array.
 
+## [0.5.2] - 2024-01-30
```

Comment:
The release should be chronologically ordered, so even if it has minor version the should be insert up to `0.6.0` version.

Reply:
Done